scripts/utils/generate_package_wiki.py

```diff
@@ -0,0 +1,88 @@
+import argparse
+import os
+import pathlib
+import xml.etree.ElementTree as ET
+from collections import defaultdict
+
+packages_by_category = defaultdict(str)
+
+
+def sort_write_wiki_content(file_path):
+    """ Writes content of packages_by_category to the wiki file
+    
+    This function retrieves rows pertaining to the key category, as a result of
+    iterating over packages_by_category.
+    Writes to the Markdown formatted string:
+    - header with the category name
+    - table with two columns: Package and Description of each of the packages
+    
+    Args:
+       file_path: path of the wiki .md file
+    """
+    wikiContent = ""
+    for category in packages_by_category.keys():
+        wikiContent += "## " + category + "\n\n"
+        wikiContent += "| Package | Description |\n"
+        wikiContent += "|---|---|\n"
```

Comment:
[nitpick] It makes it the code and the markdown a bit nicer to read:
```suggestion
        wikiContent += "| ------- | ----------- |\n"
```